.github/workflows/lint-php.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         run: composer install --prefer-dist --no-progress --no-interaction
 
       - name: Detect coding standard violations (PHPCS)
-        run: vendor/bin/phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
+        run: vendor/bin/phpcs -q --report=checkstyle --severity=1 --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
```

Comment:
Should this be 
```suggestion
        run: composer phpcs -q --report=checkstyle --runtime-set ignore_errors_on_exit 1 --runtime-set ignore_warnings_on_exit 1 | cs2pr --graceful-warnings
```

Reply:
IIRC, not using the composer script as there is additional output and differences in exit codes, breaking the `cs2pr` part.

I don't wanna risk breaking that